dotnet/src/Planners/Planners.Core/Handlebars/Extensions/HandlebarsPlannerExtensions.cs

```diff
@@ -4,7 +4,9 @@
 using System.IO;
 using System.Reflection;
 
-namespace Microsoft.SemanticKernel.Planning.Handlebars;
+#pragma warning disable IDE0130 // Namespace does not match folder structure
+namespace Microsoft.SemanticKernel.Planning.Handlebars.Extensions;
+#pragma warning restore IDE0130
```

Comment:
I'd expect the extensions to be in th same namespace as the class they are extending

Nit: there is no need to do warning restore here

Reply:
This was an intentional change by Stephen to consolidate everything into the `Planning` namespace: https://github.com/microsoft/semantic-kernel/pull/3564

And I see this error if I remove the warning ignore:
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/8c002e91-ae68-4fb1-bcbf-0ed7c1c8287e)

Reply:
But why is `.Extensions` needed?

Reply:
Ah I see what you mean, I removed it